types.go

```diff
@@ -186,4 +190,8 @@ type Orchestrator struct {
 		CertPath string `json:"cert"`
 		KeyPath  string `json:"key"`
 	} `json:"tls"`
+	Delegator struct {
+		Host string `json:"host"`
```

Comment:
this `Host` will always be the same as the Orchestrator/Nexus host, so a little redundant

Reply:
Yup good point